test/scalarstats.jl

```diff
@@ -109,6 +109,10 @@ z2 = [8. 2. 3. 1.; 24. 10. -1. -1.; 20. 12. 1. -2.]
 
 @test sem([1:5;]) ≈ 0.707106781186548
 @test sem(skipmissing([missing; 1:5; missing])) ≈ 0.707106781186548
+@test sem([1:5;], UnitWeights{Int}(5)) ≈ 0.707106781186548
+@test sem([1:5;], ProbabilityWeights([1:5;])) ≈ 0.6166 rtol=.001
+@test sem([1:100;], ProbabilityWeights([1:100;])) ≈ 2.440215 rtol=.001
```

Comment:
Can you include a test with zero weights just in case?

Reply:
Ahh, as in some of them are zero, got it.

Reply:
I mean at least having one zero.

Reply:
Added some.